opt/scopeserver/dataserver/utils/cell_color_by_features.py

```diff
@@ -75,12 +75,19 @@ def get_max_v_max(self):
     def get_cell_indices(self):
         return self.cell_indices
 
-    def normalise_vals(self, vals: np.ndarray, v_max: int) -> np.ndarray:
-        vals = np.clip(vals / v_max, None, 1)
-        vals = (
-            (constant._UPPER_LIMIT_RGB - constant._LOWER_LIMIT_RGB) * ((vals - min(vals))) / (1 - min(vals))
-        ) + constant._LOWER_LIMIT_RGB
-        vals = np.clip(vals, 0, constant._UPPER_LIMIT_RGB)
+    def normalise_vals(self, vals: np.ndarray, v_max: int, v_min: int) -> np.ndarray:
```

Comment:
This rewrite was necessary for the v_min, before things were hard coded, but remained in the scale because we had no lower limit. The rewrite prevents values below the v_min from displaying as 0's and therefore displaying no expression (grey)